### PR TITLE
fix: Encode json

### LINKED ISSUE #639: Console JSON Logging not possible
### Describe the Bug

`--encode-json` parameter is not respected

### Expected Behavior

`cloudquery fetch --enable-console-log --encode-json` should output all of the logs that went to `cloudquery.log` to stdout in a json format.

### CloudQuery Version

cloudquery version 0.22.3

### PR DESCRIPTION
**Pull request by roneli**
Closes #639

### CODE REVIEW

File: pkg/ui/output.go [hunk 1]
```diff
@@ -14,20 +15,26 @@ import (
...
+var emojiStatus = []string{color.GreenString("✓"), "📋", color.RedString("❌"), "⚠️", "⌛"}
+
 // ColorizedOutput outputs a colored message directly to the terminal.
 // The remaining arguments should be interpolations for the format string.
 func ColorizedOutput(c *color.Color, msg string, values ...interface{}) {
 	if viper.GetBool("enable-console-log") {
 		// Print output to log
-		if logMsg := strings.ReplaceAll(msg, "\n", ""); logMsg != "" {
-			log.Info().Msgf(logMsg, values...)
+		if logMsg := strip(fmt.Sprintf(msg, values...)); logMsg != "" {
```
Comment (erezrokah) [inline pkg/ui/output.go:26]: [non blocking] Can you explain the reason we need to strip emojis in the console (maybe good to comment it)?

Also, we could consider using https://github.com/acarl005/stripansi/blob/5a71ef0e047df0427e87a79f27009029921f1f9b/stripansi.go#L7 or https://github.com/chalk/ansi-regex/blob/02fa893d619d3da85411acc8fd4e2eea0e95a9d9/index.js#L3

**Review by erezrokah — APPROVED** [review]
Looks good @roneli. Left a non blocking commnet

File: pkg/ui/output.go (hunk 1 above)
Comment (roneli) [reply pkg/ui/output.go:26]: I tried them they didn't work cleanly for some reason

File: internal/logging/logger.go [hunk 2]
```diff
@@ -25,6 +25,8 @@ type Config struct {
 	Verbose bool `hcl:"verbose,optional"`
 	// EncodeLogsAsJson makes the logging framework logging JSON
 	EncodeLogsAsJson bool `hcl:"encode_logs_as_json,optional"`
+	// EncodeConsoleAsJson makes the logging framework logging JSON
```
[the remark this answers is not in the document]
Comment (roneli) [reply internal/logging/logger.go:28]: removed

**Pull request merged**